live-assignments/century-finder/answer/spec/century.test.js

```diff
@@ -0,0 +1,13 @@
+const CenturyFinder = require("../app/js/century.js");
+const centuryFinder = new CenturyFinder();
+
+test("should correctly determine corresponding century for date", function() {
+	expect(centuryFinder.findCenturyOf(new Date())).toEqual("21st");
```

Comment:
Although this test, will succeed most of the time, it can fail in some scenarios (e.g. the system where it runs has a misconfigured date). I would argue that it's better not to have it.